styles.css

```diff
@@ -117,11 +119,360 @@
+  height: 100vh;
+  margin-bottom: 30px;
+  margin-left: 1rem;
+}
+
+.about {
+  width: 30px;
+  height: 88px;
+  margin-left: 1rem;
+  margin-bottom: 10px;
+  margin-top: 50px;
+  font-size: 40px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 900;
+  color: #172b4d;
+}
+
+.experience {
+  margin-left: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-size: 19px;
+  font-weight: thin;
+  margin-bottom: 10px;
+}
+
+.connect-1 {
+  margin-left: 1rem;
+  height: 50px;
+  color: #7f8cff;
+  font-size: 20px;
+}
+
+.icons-1 {
+  height: 50px;
+  display: flex;
+  justify-content: space-evenly;
+  list-style-type: none;
+  margin-left: 1rem;
+  margin-right: 80px;
+}
+
+.icons-2 {
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.resume {
+  margin-bottom: 18px;
+  margin-left: 1rem;
+  border-radius: 8px;
+  font-size: 19px;
+  width: 150px;
+  height: 50px;
+  border: 1px solid blue;
+  background-color: #fff;
+  color: #396df2;
+}
+
+.resume:hover {
+  background-color: #afc1f0;
+}
+
+.languages-1 {
+  margin-left: 3rem;
+  color: #000;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 200;
+}
+
+.language-icons {
+  height: 200px;
+  margin-left: 3rem;
+  list-style-type: none;
+  text-align: center;
+}
+
+.button-1 {
+  border-radius: 5px;
+  width: 47vh;
+  border: 1px solid white;
+  font-size: 18px;
+}
+
+.button-2 {
+  border-radius: 8px;
+  width: 47vh;
+  border: 1px solid white;
+  font-size: 15px;
+}
+
+.button-3 {
+  border-radius: 8px;
+  width: 47vh;
+  border: 1px solid white;
+  font-size: 15px;
+}
+
+#button-12 {
+  height: 54px;
+  position: left;
+  background-image: url("images/Ellipse 1.svg");
+  background-repeat: no-repeat;
+}
+
+#button-23 {
+  height: 55px;
+  position: left;
+  background-image: url("images/Ellipse 1 (1).svg");
+  background-repeat: no-repeat;
+}
+
+#button-34 {
+  height: 55px;
+  position: left;
+  background-image: url("images/Ellipse 1 (3).svg");
+  background-repeat: no-repeat;
+}
+
+.frame {
+  font-family: 'Poppins', sans-serif;
+  margin-left: 3rem;
+  font-size: 25px;
+  height: 40px;
+  border-bottom: 1px solid rgb(186, 183, 183);
+}
+
+#frame-1 {
+  background-image: url('images/Union.svg');
+  background-repeat: no-repeat;
+  background-position: 15rem;
+}
+
+#skill-1 {
+  background-image: url('images/Union.svg');
+  background-repeat: no-repeat;
+  background-position: 15rem;
+}
+
+.skills {
+  font-family: 'Poppins', sans-serif;
+  margin-left: 3rem;
+  font-size: 25px;
+  height: 50px;
+  margin-bottom: 15px;
+  border-bottom: 1px solid rgb(186, 183, 183);
+}
+
+/* .works-container {
+  width: 95%;
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+} */
+
+.work-container {
+  display: grid;
+  grid-template-rows: 1fr;
+  min-height: 100%;
+}
+
 @media screen and (max-width: 375px) {
   body {
-    background-image: url('images/header-background-mobile.png');
+    background-image: url('images/header-background-mobile.svg');
```

Comment:
- [ ] I recommend to use an independant background for each card to look similar to the Figma template 👇🏽 
- [ ] We don't need the media queries at the moment, please remove it to avoid conflicts later.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/220786954-1598e2ba-5ab3-4da2-9f4a-e959f90d8951.png) | ![image](https://user-images.githubusercontent.com/58642949/220786825-fe833c68-3a80-4960-ac52-edc38dc5cc63.png) |